packages/esm-ward-app/src/hooks/useLocations.ts

```diff
@@ -0,0 +1,64 @@
+import { type FetchResponse, fhirBaseUrl, openmrsFetch } from '@openmrs/esm-framework';
+import { useEffect, useMemo, useState } from 'react';
+import useSWRImmutable from 'swr/immutable';
+
+interface FhirLocation {
```

Comment:
Side note, really too bad this isn't already a type in the core or in the framework